src/main/java/view/HomeView.java

```diff
@@ -25,28 +31,26 @@
     final int logoWidth = 480;
     final int logoHeight = 128;
 
-    final int gridHGap = 75;
-    final int gridVGap = 50;
     final String buttonColor = "#9CC2D4";
     final String hoverButtonColor = "#E4F6FF";
 
-    final String logoutButtonColor = "#FFFFFF";

```

Comment:
get the coolers using ThemeController!!

ThemeController themeController = ThemeController.getInstance();
Theme theme = themeController.getTheme();

With theme you can use getters like theme.getPrimaryColor()

See Theme Model!